src/views/CreateShareStructure.vue

```diff
@@ -1,18 +1,346 @@
+        the shareholder a say in how the company is being run. For most small companies starting out, a simple share
+        structure with just one class of shares (and no series) is typical.
+      </p>
+      <p>
+        The sample below is for a typical share structure with just one class of shares. If there is more than one class
+        of shares, each class must be assigned an identifying name such as Class A, Class B, etc.
+      </p>
+      <div id="share-structure-sample">
+        <ListShareClass :shareClasses="sharesHelpSample"/>
+      </div>
+      <p><small>Sample Share Structure</small></p>
+      <h4>Important Information About the Company's Share Structure</h4>
+      <p>
+        The staff at the Corporate Registry cannot provide advice on how to set up your company's share structure. If
+        you do not understand what an authorized share structure is or what its purpose is or believe you need a more
+        complex share structure, you should seek professional advice or purchase an incorporation guide for detailed
+        information and infrastructure on establishing an authorized share structure.
+      </p>
+      <p>Refer to <a href="">helpLinkGoesHere</a> to obtain more information on incorporating a company</p>
+      <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
     </section>
+
+    <p>Your application must include the following:</p>
+    <ul>
+      <li>
+        <v-icon color="blue" v-if="shareClasses.length > 0" class="cp-valid">mdi-check</v-icon>
+        <span v-else><v-icon color="red" v-if="showErrors" class="cp-invalid">mdi-close</v-icon></span>
+        <span class="chk-list-item-txt">At least one Class of Shares</span>
+      </li>
+    </ul>
+    <div class="btn-panel">
+      <v-btn outlined color="primary"  id="btn-start-add-cp"
+      :disabled="showShareStructureForm"
+      @click="initNewShareClass()" >
+        <v-icon>mdi-plus</v-icon>
+        <span>Add Share Class</span>
+      </v-btn>
+    </div>
+    <v-card flat class="add-share-structure-container" v-if="showShareStructureForm">
+      <ShareStructure
+        v-show="showShareStructureForm"
+        :initialValue="currentShareStructure"
+        :activeIndex="activeIndex"
+        :nextId="nextId"
+        :parentIndex="parentIndex"
+        :shareClasses="shareClasses"
+        @addEditClass="addEditShareClass($event)"
+        @addEditSeries="addEditShareSeries($event)"
+        @removeClass="removeShareClass($event)"
+        @removeSeries="removeSeries"
+        @resetEvent="resetData()"/>
+    </v-card>
+    <ListShareClass v-if="this.shareClasses.length > 0"
+    :shareClasses="shareClasses"
+    :componentDisabled="showShareStructureForm"
+    @editClass="initShareClassForEdit($event)"
+    @removeClass="removeShareClass($event)"
+    @addSeries="initNewShareSeries($event)"
+    @editSeries="editSeries"
+    @removeSeries="removeSeries"/>
```

Comment:
To be consistent with existing code style, please format this as follows:
```
<ListShareClass ...
   ...
   ...
/>
```

Reply:
Fixed